From 8a038a4430107856273743655b1802fe64cb68c3 Mon Sep 17 00:00:00 2001
From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Date: Mon, 14 Nov 2016 08:16:50 +1100
Subject: [PATCH 09/12] dpnet: Header files cleanup

Also stops error of SOCKADDR being redefined on older compiliers

Signed-off-by: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
---
 dlls/dpnet/address.c    | 1 -
 dlls/dpnet/client.c     | 1 -
 dlls/dpnet/lobbiedapp.c | 2 --
 dlls/dpnet/peer.c       | 1 -
 dlls/dpnet/server.c     | 1 -
 dlls/dpnet/threadpool.c | 1 -
 include/dpaddr.h        | 2 ++
 7 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/dlls/dpnet/address.c b/dlls/dpnet/address.c
index c5457eb..4d55caa 100644
--- a/dlls/dpnet/address.c
+++ b/dlls/dpnet/address.c
@@ -33,7 +33,6 @@
 #include "wine/unicode.h"
 #include "wine/debug.h"
 
-#include "dplay8.h"
 #include "dpnet_private.h"
 
 WINE_DEFAULT_DEBUG_CHANNEL(dpnet);
diff --git a/dlls/dpnet/client.c b/dlls/dpnet/client.c
index 4f272d6..27a430e 100644
--- a/dlls/dpnet/client.c
+++ b/dlls/dpnet/client.c
@@ -31,7 +31,6 @@
 #include "objbase.h"
 #include "wine/debug.h"
 
-#include "dplay8.h"
 #include "dpnet_private.h"
 
 WINE_DEFAULT_DEBUG_CHANNEL(dpnet);
diff --git a/dlls/dpnet/lobbiedapp.c b/dlls/dpnet/lobbiedapp.c
index 8ec9ad1..d041e63 100644
--- a/dlls/dpnet/lobbiedapp.c
+++ b/dlls/dpnet/lobbiedapp.c
@@ -31,8 +31,6 @@
 #include "objbase.h"
 #include "wine/debug.h"
 
-#include "dplay8.h"
-#include "dplobby8.h"
 #include "dpnet_private.h"
 
 WINE_DEFAULT_DEBUG_CHANNEL(dpnet);
diff --git a/dlls/dpnet/peer.c b/dlls/dpnet/peer.c
index 5b4cb4f..b12cfb9 100644
--- a/dlls/dpnet/peer.c
+++ b/dlls/dpnet/peer.c
@@ -33,7 +33,6 @@
 #include "objbase.h"
 #include "wine/debug.h"
 
-#include "dplay8.h"
 #include "dpnet_private.h"
 
 WINE_DEFAULT_DEBUG_CHANNEL(dpnet);
diff --git a/dlls/dpnet/server.c b/dlls/dpnet/server.c
index d91822e..46d5fe2 100644
--- a/dlls/dpnet/server.c
+++ b/dlls/dpnet/server.c
@@ -32,7 +32,6 @@
 #include "ocidl.h"
 #include "wine/debug.h"
 
-#include "dplay8.h"
 #include "dpnet_private.h"
 
 typedef struct IDirectPlay8ServerImpl
diff --git a/dlls/dpnet/threadpool.c b/dlls/dpnet/threadpool.c
index 01ad73d..0f4e108 100644
--- a/dlls/dpnet/threadpool.c
+++ b/dlls/dpnet/threadpool.c
@@ -32,7 +32,6 @@
 #include "objbase.h"
 #include "wine/debug.h"
 
-#include "dplay8.h"
 #include "dpnet_private.h"
 
 WINE_DEFAULT_DEBUG_CHANNEL(dpnet);
diff --git a/include/dpaddr.h b/include/dpaddr.h
index 576caaf..d47dd26 100644
--- a/include/dpaddr.h
+++ b/include/dpaddr.h
@@ -27,7 +27,9 @@ extern "C" {
 #endif /* defined(__cplusplus) */
 
 typedef REFIID	        DPNAREFIID;
+#ifndef _WINSOCK2API_
 typedef struct sockaddr SOCKADDR;
+#endif
 
 /*****************************************************************************
  * DirectPlay8Addr defines
-- 
2.10.2

